contracts/Vault.vy

```diff
@@ -476,7 +476,7 @@ def setManagementFee(fee: uint256):
     @param fee The new management fee to use.
     """
     assert msg.sender == self.governance
-    assert fee <= MAX_BPS / 2
+    assert fee <= MAX_BPS
```

Comment:
@zgfzgf this one doesn't have to be divided by 2, can you update the tests so that it's calling setManagementFee with `MAX_BPS`

Reply:
OK. I think commit code.
Later change tests file. because it's not easy to merge the code here. It's better to reopen an issue
may you update tests file?

Reply:
Which program do you use for git management?
It should be fairly easy for you to pull the latest changes and update the tests.

Reply:
I see.I delete you code.I update it and update test.
later check ok

Reply:
you can just pull my change with `git pull`.

Reply:
haha. I see.